notebooks/en/agent_rag.ipynb

```diff
@@ -0,0 +1,2800 @@
+{
```

Comment:
<p>maybe explain why you picked thenlper/gte-small?</p><p>the rest is self explanatory for someone who read your previous cookbook I guess</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/135/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>